core/sequencer/sequencer.go

```diff
@@ -223,25 +231,29 @@ func (s *Sequencer) applyMutations(mutations []*mutator.QueueMessage, leaves []*
 			oldValue, err = entry.FromLeafValue(leaf.GetLeafValue())
 			if err != nil {
 				glog.Warningf("entry.FromLeafValue(%v): %v", leaf.GetLeafValue(), err)
+				mutationFailures.Inc(d.DomainID, "Unmarshal")
 				continue
 			}
 		}
 
 		newValue, err := s.mutatorFunc.Mutate(oldValue, m.Mutation)
 		if err != nil {
 			glog.Warningf("Mutate(): %v", err)
+			mutationFailures.Inc(d.DomainID, "Mutate")
 			continue // A bad mutation should not make the whole batch fail.
 		}
 		leafValue, err := entry.ToLeafValue(newValue)
 		if err != nil {
 			glog.Warningf("ToLeafValue(): %v", err)
+			mutationFailures.Inc(d.DomainID, "Marshal")
 			continue
 		}
 
 		// Serialize commitment.
 		extraData, err := proto.Marshal(m.ExtraData)
 		if err != nil {
 			glog.Warningf("Marshal(committed proto): %v", err)
+			mutationFailures.Inc(d.DomainID, "Marshal")
```

Comment:
Is it worth distinguishing `LeafMarshal` and `ProtoMarshal`?

Reply:
I don't think so.  Marshaling errors should be extremely rare.